ui/app/routes/vault/cluster/access/mfa/index.js

```diff
@@ -0,0 +1,37 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+export default class MfaRoute extends Route {
+  @service router;
+
+  queryParams = {
+    page: {
+      refreshModel: true,
+    },
+  };
+
+  model(params) {
+    return this.store
+      .lazyPaginatedQuery('mfa-method', {
+        responsePath: 'data.keys',
+        page: params.page || 1,
+      })
+      .then((model) => {
+        return model;
+      })
+      .catch((err) => {
+        if (err.httpStatus === 404) {
+          return [];
+        } else {
+          throw err;
+        }
+      });
+  }
+  afterModel(model) {
+    if (model.get('length') === 0) {
+      this.router.transitionTo('vault.cluster.access.mfa.configure');
+    }
```

Comment:
Nice redirect to the landing page here! Just a nit on using `get`. You can just do `model.length`.